test/flow.ts

```diff
@@ -1042,37 +1041,34 @@ test('idp sends a redirect logout request with signature and sp parses it', asyn
   const octetString = Object.keys(originalURL.query).map(q => q + '=' + encodeURIComponent(originalURL.query[q] as string)).join('&');
   const { extract } = await spWantLogoutReqSign.parseLogoutRequest(idp, 'redirect', { query: { SAMLRequest, Signature, SigAlg }, octetString});
   t.is(extract.nameID, 'user@esaml2.com');
-  t.is(extract.sessionIndex, '_664ade6a050f55a2c7cb2fb0571df7280365c0c7')
   t.is(extract.issuer, 'https://idp.example.com/metadata');
   t.is(typeof extract.request.id, 'string');
   t.is(extract.request.destination, 'https://sp.example.org/sp/slo');
   t.is(extract.signature, null); // redirect binding doesn't embed the signature
 });
 
 test('idp sends a post logout request without signature and sp parses it', async t => {
-  const { relayState, type, entityEndpoint, id, context } = idp.createLogoutRequest(sp, 'post', { logoutNameID: 'user@esaml2.com', sessionIndex: '_664ade6a050f55a2c7cb2fb0571df7280365c0c7' }) as PostBindingContext;
+  const { type, entityEndpoint, id, context } = idp.createLogoutRequest(sp, 'post', { logoutNameID: 'user@esaml2.com' }) as PostBindingContext;
   t.is(typeof id, 'string');
   t.is(typeof context, 'string');
   t.is(typeof entityEndpoint, 'string');
   t.is(type, 'SAMLRequest');
   const { extract } = await sp.parseLogoutRequest(idp, 'post', { body: { SAMLRequest: context } });
   t.is(extract.nameID, 'user@esaml2.com');
-  t.is(extract.sessionIndex, '_664ade6a050f55a2c7cb2fb0571df7280365c0c7')
```

Comment:
any reason to remove the session index

Reply:
If you note the commit where these were removed: https://github.com/tngan/samlify/pull/511/commits/5c78b94f05962d01f0f526907f90743e57809646, these tests never passed since the sessionIndex was added. The problem is that `sessionIndex` is not part of the default logout request template, so there is nothing for the extractor to find.

Reply:
thanks for clarify